packages/widget-message-meet/src/containers/scrolling-activity/__snapshots__/index.test.js.snap

```diff
@@ -4,200 +4,7 @@ exports[`ScrollingActivity container renders properly 1`] = `
-              </div>
-            </div>
-            <div
-              className="activity-content">
-              <div
-                className="activity-text"
-                dangerouslySetInnerHTML={
-                  Object {
-                    "__html": "Test Activity Content 2",
-                  }
-                } />
-            </div>
-          </div>
-          <div
-            className="activity-post-actions">
-            <div
-              className="activity-post-action">
-              <div
-                className="post-action-item">
-                <button
-                  className="button"
-                  onClick={[Function]}
-                  onKeyUp={[Function]}>
-                  <div
-                    className="icon"
-                    title="Flag this message" />
-                </button>
-              </div>
-            </div>
-            <div
-              className="activity-post-action">
-              <div
-                className="action-spacer" />
-            </div>
-          </div>
-        </div>
-      </div>
-      <div
-        className="activity-item-container">
-        <div
-          className="activity-item">
-          <div
-            className="avatar-wrapper">
-            <div
-              className="avatar"
-              style={undefined}>
-              <span
-                className="avatar-letter">
-                T
-              </span>
-            </div>
-          </div>
-          <div
-            className="content-container">
-            <div
-              className="meta">
-              <div
-                className="display-name"
-                title="{name}">
-                Test User 3
-              </div>
-              <div
-                className="published">
-                9/20/16, 3:54 PM
-              </div>
-            </div>
-            <div
-              className="activity-content">
-              <div
-                className="activity-text"
-                dangerouslySetInnerHTML={
-                  Object {
-                    "__html": "Test Activity Content 3",
-                  }
-                } />
-            </div>
-          </div>
-          <div
-            className="activity-post-actions">
-            <div
-              className="activity-post-action">
-              <div
-                className="post-action-item">
-                <button
-                  className="button"
-                  onClick={[Function]}
-                  onKeyUp={[Function]}>
-                  <div
-                    className="icon"
-                    title="Flag this message" />
-                </button>
-              </div>
-            </div>
-            <div
-              className="activity-post-action">
-              <div
-                className="action-spacer" />
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
+      className="activity-list" />
```

Comment:
Looks like the activities are missing here as well. Is this because of a shallow render or just that the selector isn't firing correctly?